src/libs/OptionsListUtils.ts

```diff
@@ -1472,7 +1476,12 @@ function getOptions(
         const isTaskReport = ReportUtils.isTaskReport(report);
         const isPolicyExpenseChat = ReportUtils.isPolicyExpenseChat(report);
         const isMoneyRequestReport = ReportUtils.isMoneyRequestReport(report);
-        const accountIDs = report.visibleChatMemberAccountIDs ?? [];
+        const isSelfDM = ReportUtils.isSelfDM(report);
+        let accountIDs = report.visibleChatMemberAccountIDs ?? [];
+
+        if (isSelfDM) {
+            accountIDs = [currentUserAccountID ?? 0];
+        }
```

Comment:
NAB, but suggestion to keep `const`:
```suggestion
        const accountIDs = isSelfDM ? [currentUserAccountID ?? 0] : report.visibleChatMemberAccountIDs ?? [];
```